apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx

```diff
@@ -76,3 +77,25 @@ export const SiteDefinition: ISiteDefinition<Platforms> = {
     '#/styles/utilities': '#/styles/web'
   }
 };
+
+function _renderSiteMessageBar(pagePath?: string): JSX.Element | undefined {
```

Comment:
This seems a lot more awkward and hack-ish than having Site render the message bar. A somewhat better design might be to put the URLs that should show a message bar and the corresponding props in the site definition:
```ts
// (I don't like this interface name but can't think of a better one right now.)
interface IMessageBarConfig {
  url: string | RegExp;
  messageBarProps: ISiteMessageBarProps;
}

interface ISiteDefinition {
  /**
   * If the route path matches any of the strings or regular expressions specified in
   * an entry's `url` property, show a message bar with the corresponding props.
   */
  messageBars?: IMessageBarConfig[];
}
```

Reply:
It is a little hacky, I did a little exploration of just defining the props in an object per page, but it was going to take some refactoring of getting the page props in `Site` in order for the `renderSiteMessageBar` method to get those props from sub pages. It worked on top level pages like controls and home page fine though.